src/index.css

```diff
@@ -1,206 +1,6 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-family: 'Gill Sans', sans-serif;
 }
-
-:root {
-  --purple: rgb(128, 0, 128);
-  --white: #fff;
-  --darkpurple: #fd33a2;
-}
-
-body {
-  background-color: var(--purple);
-}
-
-li {
-  list-style: none;
-}
-
-.App {
-  background-color: var(--purple);
-}
-
-.home-list {
-  display: grid;
-  grid-template-columns: 1fr  1fr;
-}
-
-.columns {
-  display: flex;
-  align-items: center;
-  background-color: var(--purple);
-  padding: 1rem 0;
-  padding-left: 0.5rem;
-}
-
-.columns:focus {
-  border: 1px solid var(--white);
-}
-
-.columns:nth-child(4n),
-.columns:nth-child(4n+1) {
-  background-color: var(--darkpurple);
-}
-
-a {
-  text-decoration: none;
-  color: var(--white);
-}
-
-.home-case-names {
-  font-weight: 700;
-  font-size: 1rem;
-  margin-bottom: 0.5rem;
-}
-
-.home-case-numbers {
-  font-size: 0.7rem;
-}
-
-.home-header {
-  display: flex;
-  align-items: center;
-  gap: 3rem;
-  background-color: var(--purple);
-  justify-content: center;
-  padding: 4rem 0;
-}
-
-.home-header h1 {
-  color: var(--white);
-}
-
-.search-bar {
-  background-color: var(--purple);
-  text-align: center;
-}
-
-nav {
-  background-color: var(--darkpurple);
-  width: 100%;
-}
-
-nav a {
-  padding: 1rem;
-  display: flex;
-  gap: 3rem;
-}
-
-.header-title {
-  font-weight: 700;
-}
-
-input {
-  border: none;
-  border-bottom: 1px solid var(--darkpurple);
-  font-size: 1rem;
-  padding: 0.5rem;
-  width: 100%;
-}
-
-input:focus {
-  outline: none;
-  border: 1px solid black;
-}
-
-.dot-shuttle {
-  position: relative;
-  left: 160px;
-  width: 12px;
-  height: 12px;
-  border-radius: 6px;
-  background-color: black;
-  color: transparent;
-  margin: 7px 0;
-}
-
-.dot-shuttle::before,
-.dot-shuttle::after {
-  content: '';
-  display: inline-block;
-  position: absolute;
-  top: 0;
-  width: 12px;
-  height: 12px;
-  border-radius: 6px;
-  background-color: black;
-  color: transparent;
-}
-
-.dot-shuttle::before {
-  left: 15px;
-  animation: dotShuttle 2s infinite ease-out;
-}
-
-.dot-shuttle::after {
-  left: 30px;
-}
-
-@keyframes dotShuttle {
-  0%,
-  50%,
-  100% {
-    transform: translateX(0);
-  }
-
-  25% {
-    transform: translateX(-50px);
-  }
-
-  75% {
-    transform: translateX(50px);
-  }
-}
-
-.loading-page {
-  background-color: var(--purple);
-  height: 100vh;
-  text-align: center;
-}
-
-.loading {
-  font-size: 2rem;
-  font-weight: 700;
-}
-
-.details-section {
-  background-color: var(--purple);
-}
-
-.details-title {
-  font-weight: 700;
-  font-size: 1.8rem;
-  margin-bottom: 1rem;
-  color: var(--white);
-}
-
-.details-total {
-  font-size: 1.2rem;
-  margin-bottom: 1rem;
-  color: var(--white);
-}
-
-.details-row {
-  color: var(--white);
-  padding: 1rem;
-  height: 17vh;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  font-size: 1.2rem;
-}
-
-.details-row:nth-child(odd) {
-  background-color: var(--darkpurple);
-}
-
-.details-header {
```

Comment:
from what I can see all your files have reverted and hold no current changes to the project, please fix this and request a  review again

Reply:
Hello @alexander16108 thanks for the review, I really appreciate your feedback. I mistakenly merge the pull request I was supposed to submit that was why I submitted the reverted one. My question now is that show I make a new commit and open a pull request then submit it or what do you think I should do ?